src/test/regress/sql/multi_alter_table_add_constraints_without_name.sql

```diff
@@ -785,38 +785,6 @@ SELECT con.conname
 \c - - :master_host :master_port
 ALTER TABLE AT_AddConstNoName.citus_local_partitioned_table DROP CONSTRAINT citus_local_partitioned_table_partition_col_key;
 
--- Check "ADD EXCLUDE" errors out for partitioned table since the postgres does not allow it
-ALTER TABLE AT_AddConstNoName.citus_local_partitioned_table ADD EXCLUDE(partition_col WITH =);
```

Comment:
I think that in this test, we only need to remove the line that adds the exclusion constraint, not the rest.

Reply:
Done